### PR TITLE
Fix for issue#662

### PR DESCRIPTION
**Pull request by Necravisaketi**
## Proposed changes

Added logs in src/wq/core/task.py. fix for issue #662 

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Maintenance (update of libraries or other dependences)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING][contrib] doc
- [x] I have signed the [CLA][cla]
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have run **all** the existing tests locally (not just those related to my feature) and there are no errors
- [x] After the last push to the PR branch, I have run the lint script locally and there are no changes to the code base
- [x] I have updated the [RELEASE NOTES][release]
- [ ] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

[cla]: https://raw.githubusercontent.com/orchestracities/ngsi-timeseries-api/master/individual_cla.pdf
    "Martel Open Source Software Individual Contributor License Agreement"
[contrib]: https://github.com/orchestracities/ngsi-timeseries-api/blob/master/CONTRIBUTING.md
    "Contributing to QuantumLeap"
[release]: https://github.com/orchestracities/ngsi-timeseries-api/blob/master/RELEASE_NOTES.md
    "QuantumLeap Release Notes"

### CODE REVIEW

File: src/wq/core/task.py [hunk 1]
```diff
@@ -270,10 +276,10 @@ def _do_enqueue(self):
                             result_ttl=self.success_ttl(),
                             failure_ttl=self.failure_ttl())
         except Exception as e:
-            print(e)    # TODO log error
+            log().error(e)
             if recover_from_enqueueing_failure():
-                # TODO log msg to say you'll run this task on the spot
-                # last ditch attempt, but only if configured to do so.
+                msg="you'll run this task on the spot last ditch attempt, but only if configured to do so"
```
Comment (c0c0n3) [inline src/wq/core/task.py:281]: @Necravisaketi can we change the message just slightly? Like: 

> This task could not be added to the work queue, QuantumLeap will try running this task synchronously if WQ_RECOVER_FROM_ENQUEUEING_FAILURE = true.

thanks so much!

**Comment by Necravisaketi** [comment]
Hi, @c0c0n3 @chicco785 when i run the tests locally all tests were passed but in the CircleCi the ``test_others`` tests are failing. Sometimes after running so many times, it will pass automatically.

**Comment by c0c0n3** [comment]
@Necravisaketi see my comment on #661. It seems bizarre, but it looks like adding log statements to WQ causes that error? Can't believe it, but can we try getting to the bottom of this?

**Comment by c0c0n3** [comment]
@Necravisaketi actually I'm wrong about `test_others` failures being related to this PR. If you look at #660 you'll see there's no code changes in there, but `test_others` still failed. I've opened a separate issue about it: #666. Meanwhile we can ignore the failing test here, rebase on master **after merging** #661, then merge this PR. Thanks alot!

**Comment by c0c0n3** [comment]
@Necravisaketi can you please run the linter on locally on your branch, see what files it changes and figure out what goes wrong? thanks alot, much appreciated!

**Comment by c0c0n3** [comment]
@Necravisaketi it looks like this PR is good to go? There's a couple of issues but I don't think they've got anything to do w/ your code:

* reporter test failing. we talked about this already, see my other comments. definitely not related to your code change.
* linter failing. I saw your latest commit to fix the linter issue and I think the issue is actually fixed, thanks! the problem is the github action can't run cleanly for some reason. I supposed we can ignore this.

Should we merge or would you still like to add more code?

**Comment by Necravisaketi** [comment]
Hi, @c0c0n3 I have done from my side for this issue and there are no conflicts with these changes so we need to merge this pr.

**Comment by c0c0n3** [comment]
good stuff, thanks alot @Necravisaketi!

**Pull request merged**